src/new.rs

```diff
@@ -3,12 +3,20 @@ use crate::utils::{
 };
 use fuels::prelude::*;
 use std::path::Path;
+use clap::Args;
 
-pub fn new_wallet_cli(wallet_path: &Path) -> anyhow::Result<()> {
+#[derive(Debug, Args)]
+pub struct New {
+    /// Set true to automatically replace the existing wallet
+    #[clap(short, long)]
+    force: bool,
+}
+
+pub fn new_wallet_cli(wallet_path: &Path, new: New) -> anyhow::Result<()> {
     let password = request_new_password();
```

Comment:
Perhaps we should check for the existing wallet first, before asking for a password.